.dotnet/src/Custom/OpenAIClient.cs

```diff
@@ -237,11 +237,9 @@ internal static ClientPipeline CreatePipeline(ApiKeyCredential credential, OpenA
             perCallPolicies: [
                 CreateAddBetaFeatureHeaderPolicy(),
                 CreateAddCustomHeadersPolicy(options),
-            ],
-            perTryPolicies:
-            [
                 ApiKeyAuthenticationPolicy.CreateHeaderApiKeyPolicy(credential, AuthorizationHeader, AuthorizationApiKeyPrefix)
             ],
+            perTryPolicies: [],
```

Comment:
In your description, it says: "Move the authorization header policy application to per-try", but this change looks like it is making it per-call. Am I misreading it?

Reply:
Eep! You're not misreading -- I should change the description since this is *very* confusing. The key distinction is "before retry" vs. "after retry," and I inverted the two since that maps somewhat strangely.

Reply:
Backed that out since it shouldn't be relevant. The root cause of the observed 401 issue on Azure is almost certainly just the retry being done too quickly -- so I scoped this to just adding in the header checks.